<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>SAKSerialPortToolUi</class>
 <widget class="QWidget" name="SAKSerialPortToolUi">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>171</width>
    <height>186</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string/>
  </property>
  <layout class="QGridLayout" name="gridLayout">
   <property name="leftMargin">
    <number>0</number>
   </property>
   <property name="topMargin">
    <number>0</number>
   </property>
   <property name="rightMargin">
    <number>0</number>
   </property>
   <property name="bottomMargin">
    <number>0</number>
   </property>
   <item row="0" column="1">
    <widget class="SAKPortNameComboBox" name="comboBoxPortNames">
     <property name="editable">
      <bool>false</bool>
     </property>
    </widget>
   </item>
   <item row="0" column="0">
    <widget class="QLabel" name="label_2">
     <property name="sizePolicy">
      <sizepolicy hsizetype="Fixed" vsizetype="Preferred">
       <horstretch>0</horstretch>
       <verstretch>0</verstretch>
      </sizepolicy>
     </property>
     <property name="text">
      <string>Port name</string>
     </property>
    </widget>
   </item>
   <item row="2" column="1">
    <widget class="SAKDataBitsComboBox" name="comboBoxDataBits"/>
   </item>
   <item row="3" column="1">
    <widget class="SAKStopBitsComboBox" name="comboBoxStopBits"/>
   </item>
   <item row="4" column="1">
    <widget class="SAKParityComboBox" name="comboBoxParity"/>
   </item>
   <item row="1" column="1">
    <widget class="SAKBaudRateComboBox" name="comboBoxBaudRate">
     <property name="editable">
      <bool>true</bool>
     </property>
    </widget>
   </item>
   <item row="5" column="1">
    <widget class="SAKFlowControlComboBox" name="comboBoxFlowControl"/>
   </item>
   <item row="1" column="0">
    <widget class="QLabel" name="label">
     <property name="sizePolicy">
      <sizepolicy hsizetype="Fixed" vsizetype="Preferred">
       <horstretch>0</horstretch>
       <verstretch>0</verstretch>
      </sizepolicy>
     </property>
     <property name="text">
      <string>Baud rate</string>
     </property>
    </widget>
   </item>
   <item row="2" column="0">
    <widget class="QLabel" name="label_3">
     <property name="sizePolicy">
      <sizepolicy hsizetype="Fixed" vsizetype="Preferred">
       <horstretch>0</horstretch>
       <verstretch>0</verstretch>
      </sizepolicy>
     </property>
     <property name="text">
      <string>Data bits</string>
     </property>
    </widget>
   </item>
   <item row="3" column="0">
    <widget class="QLabel" name="label_4">
     <property name="sizePolicy">
      <sizepolicy hsizetype="Fixed" vsizetype="Preferred">
       <horstretch>0</horstretch>
       <verstretch>0</verstretch>
      </sizepolicy>
     </property>
     <property name="text">
      <string>Stop bits</string>
     </property>
    </widget>
   </item>
   <item row="4" column="0">
    <widget class="QLabel" name="label_5">
     <property name="sizePolicy">
      <sizepolicy hsizetype="Fixed" vsizetype="Preferred">
       <horstretch>0</horstretch>
       <verstretch>0</verstretch>
      </sizepolicy>
     </property>
     <property name="text">
      <string>Parity</string>
     </property>
    </widget>
   </item>
   <item row="5" column="0">
    <widget class="QLabel" name="label_6">
     <property name="sizePolicy">
      <sizepolicy hsizetype="Fixed" vsizetype="Preferred">
       <horstretch>0</horstretch>
       <verstretch>0</verstretch>
      </sizepolicy>
     </property>
     <property name="text">
      <string>Flow control</string>
     </property>
    </widget>
   </item>
  </layout>
 </widget>
 <customwidgets>
  <customwidget>
   <class>SAKPortNameComboBox</class>
   <extends>QComboBox</extends>
   <header location="global">SAKPortNameComboBox.hh</header>
  </customwidget>
  <customwidget>
   <class>SAKBaudRateComboBox</class>
   <extends>QComboBox</extends>
   <header location="global">SAKBaudRateComboBox.hh</header>
  </customwidget>
  <customwidget>
   <class>SAKDataBitsComboBox</class>
   <extends>QComboBox</extends>
   <header location="global">SAKDataBitsComboBox.hh</header>
  </customwidget>
  <customwidget>
   <class>SAKStopBitsComboBox</class>
   <extends>QComboBox</extends>
   <header location="global">SAKStopBitsComboBox.hh</header>
  </customwidget>
  <customwidget>
   <class>SAKParityComboBox</class>
   <extends>QComboBox</extends>
   <header location="global">SAKParityComboBox.hh</header>
  </customwidget>
  <customwidget>
   <class>SAKFlowControlComboBox</class>
   <extends>QComboBox</extends>
   <header location="global">SAKFlowControlComboBox.hh</header>
  </customwidget>
 </customwidgets>
 <resources/>
 <connections/>
</ui>
